hydromt_delft3dfm/utils.py

```diff
@@ -957,7 +953,6 @@ def write_1dlateral(
                 ext = dict()
                 ext["id"] = latid
                 ext["name"] = latid
-                ext["quantity"] = "discharge"
```

Comment:
Again I unfortunately don't know enough, but @shartgring can test if the lateral inflow still work without this.

Reply:
The same reason, this is a `[Lateral]` block that does not have a quantity parameter, also not according to the manual. But indeed, it would be a very good idea to test if the resulting models still work.